src/Perspex.Controls/MenuItem.cs

```diff
@@ -303,11 +303,30 @@ protected override void OnPointerEnter(PointerEventArgs e)
                     IsSubMenuOpen = true;
                 }
             }
-            else if (HasSubMenu && !IsSubMenuOpen)
+            else
             {
-                _submenuTimer = DispatcherTimer.Run(
-                    () => IsSubMenuOpen = true,
-                    TimeSpan.FromMilliseconds(400));
+                var parentItem = Parent as MenuItem;
+                if (parentItem != null)
+                {
+                    foreach (var child in Parent.LogicalChildren.OfType<MenuItem>())
+                    {
+                        if (child != this && child.IsSubMenuOpen)
+                        {
+                            child.IsSubMenuOpen = false;
+                            child.IsSelected = child.AlwaysSelected;
```

Comment:
I think this can just be `child.IsSelected = false` as `AlwaysSelected` will never be true on a `MenuItem`.